index.html

```diff
@@ -8,9 +8,42 @@
     <link rel="stylesheet"  href="style.css">
 </head>
 <body>
-    <header>
-        
+    <div class ="porfolio_container">
+    
+    <header >
+        <nav >
+            <ul class ="top_navigation">
+                <li class ="nameplate"> 9:41</li>
+                <ui class ="nameplateicons">
+                <li class ="icons"> <img src = "image/Cellular Connection.png"></li>
+                <li class ="icons"> <img src = "image/Wifi.png"></li>
+                <li class ="icons"> <img src = "image/Battery.png"></li>
+            </ui>
+        </nav>
+        <nav >
+            <ul class ="item_navigation">
+                <li class ="item"><span class= "text"><h2>Omotayo Akande</h2></span></li>
+                <li class ="union"><img src = "image/Union.png"></li>
+               
+            </ul>
+    
+    
+        </nav>
     </header>
     
+    <section >
+        <div class ="headline">
+        <h1 class = "primary">Hey there<br>I’m Tayo.</h1> 
+        <h2> I am a Software developer</h2>
+            
+        <h3 class="supportingtext"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</h3>
```

Comment:
- [ ] According to the project requirements, this text is to be enclosed in a (`p` tag) and not an (`h3` tag) :100: doing this makes your page  semantically correct :wink: